generate_test.go

```diff
@@ -27,8 +27,9 @@ message test {
   required float f32 = 11;
   required double f64 = 12;
   required bytes bytes = 13;
-  required string string = 14;
-  required emb struct = 15;
+  required bytes array = 14;
+  required string string = 15;
+  required emb struct = 16;
```

Comment:
More specific: I think the correct 'proto'-file needs sth similar to:
```
message emb {
   required Id int; 
   // ...
}
```
before the usage of `emb` to be a valid proto file